README.md

```diff
@@ -40,40 +39,8 @@ virtualenv -p python3 venv
 pip install -e .[dev]
 ```
 
-### Testing Setup
```

Comment:
I'm curious about this section removal - do you know if this still works?

Reply:
I don't, didn't test. Just didn't seem appropriate/necessary to have personal dev preferences here.